### PR TITLE
Ability to get travel and insurance certificates

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

- Add insurance certificate links
- Add travel certificate integration that uses default data and blows up for non-Home contracts

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3336** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3336" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3336?utm_source=stack-comment).

File: apps/store/src/features/memberArea/components/TravelCertificateButton.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  const { data } = useMemberAreaMemberInfoQuery()
+  const [sendRequest, result] = useTravelCertificateCreateMutation()
+
+  const handleRequestCertificate = useCallback(() => {
+    alert(
+      `Demo version!
+      
+      In real life there should be dialog asking you for certificate parameters. We will use defaults for now`,
+    )
+    const todayDate = new Date().toISOString().split('T')[0]
```
Comment (robinandeer) [inline apps/store/src/features/memberArea/components/TravelCertificateButton.tsx:21]: we might have a utility for this... like `formatAPIDate`

**Review by robinandeer — APPROVED** [review]
Awesome stuff -- just added some refactor comments

**Review by gustaveen — APPROVED** [review]
Nice, let's move it from the InsuranceCards though when we have specific insurance page 👍

**Comment by alebedev** [comment]
### Merge Activity

* **Oct 5, 4:58 AM**: @alebedev merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3336).

**Pull request merged**